src/transformers/models/tvp/modeling_tvp.py

```diff
@@ -576,6 +626,9 @@ def _init_weights(self, module):
 
         return_dict (`bool`, *optional*):
             Whether or not to return a [`~utils.ModelOutput`] instead of a plain tuple.
+
+        interpolate_pos_encoding (`bool`, *optional*, defaults to `False`):
+            Whether to interpolate the pre-trained image pad prompter encodings.
```

Comment:
This should refer to the positional and pad encoding here. 

Normally, I'd say we should have the input argument reflect the behaviour better e.g. `interpolate_encoding`. However, it's probably better to be consistent across the models' forward passes for the main class.